README.md

```diff
@@ -101,28 +101,23 @@ With the `updateId`, you can check the status (`enqueued`, `processed` or `faile
 
 #### Basic Search <!-- omit in toc -->
 
-A basic search can be performed by calling the `index.search()` method, with a simple string query.
+A basic search can be performed by calling `index.search()` method, with a simple string query.
 
 ```java
+import com.meilisearch.sdk.model.SearchResult;
+
 // MeiliSearch is typo-tolerant:
-String results = index.search("harry pottre");
-System.out.println(results);
+SearchResult results = index.search("harry pottre");
+System.out.println(results.getOffset());
+System.out.println(results.getLimit());
+System.out.println(results.getQuery());
 ```
 
-Output:
-```json
-{
-	"hits": [{
-		"book_id": 4,
-		"title": "Harry Potter and the Half-Blood Prince"
-	}],
-	"offset": 0,
-	"limit": 20,
-	"nbHits": 1,
-	"exhaustiveNbHits": false,
-	"processingTimeMs": 2,
-	"query": "harry pottre"
-}
+the result will be:
+```
+0
+20
+harry pottre
```

Comment:
```suggestion
- Output:
```
SearchResult(hits=[{book_id=4.0, title=Harry Potter and the Half-Blood Prince}], offset=0, limit=20, nbHits=1, exhaustiveNbHits=false, facetsDistribution=null, exhaustiveFacetsCount=false, processingTimeMs=3, query=harry pottre)
```